src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export default interface User {
```

Comment:
"default" is not necessary here. As I can understand, the interface User will always be only one in this document